lib/p2p/Pool.ts

```diff
@@ -829,6 +840,13 @@ class Pool extends EventEmitter {
 
     peer.once('close', () => this.handlePeerClose(peer));
 
+    peer.on('bannedBy', async (nodePubKey: string) => {
+      // set peer as banning us
+      await this.nodes.persistBannedBy(nodePubKey);
+      this.nodes.setBannedBy(nodePubKey);
+      this.logger.info(`Peer: ${nodePubKey} has banned you`);
```

Comment:
I think "has banned us" is more in line with how we log such messages elsewhere.